core/backend/src/CloudSqlite.ts

```diff
@@ -570,12 +570,12 @@ export namespace CloudSqlite {
     cacheDir?: string;
   }
 
-  /** The collection of currently extant CloudCaches, by name. */
+  /** The collection of currently extant `CLoudCache`s, by name. */
```

Comment:
Should be 'CloudCache' instead of 'CLoudCache'

Reply:
fixed